.github/workflows/static-site-qa-template.yml

```diff
@@ -12,12 +12,12 @@ jobs:
         uses: actions/checkout@v3
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '3.1'
+          ruby-version: '3.1.4'
```

Comment:
Are we sure we want to hardcode this to a patch version? Seems pretty specific, and will require a lot of watching and manual changes when releases happen...

Reply:
I chose 3.1.4 because it's the last patch in 3.1 and I figured it's the most mature release of 3.1? that's a fair point i'll change it back.

Reply:
changed in f4ee606f8d131f88c78e4be54787af0983a238e8